virocon/plotting.py

```diff
@@ -84,34 +84,23 @@ def get_default_semantics(n_dim):
     return semantics
 
 
-def _get_n_axes(n_intervals):
-    max_intervals = 16
-    if n_intervals > max_intervals:
-        raise NotImplementedError(
-            f"Automatic axes creation is only supported for up to {max_intervals} intervals."
+def _get_n_axes(n_intervals, max_cols=4):
```

Comment:
Comparing the new `_get_n_axes` with the old version, I found a "bug" in the old version, which still persists in the updated version. When `n_intervals=0` both versions return (1,1) axes, which is unexpected. The old version also had an offset by 1. 

```suggestion
def _get_n_axes(n_intervals, max_cols=4):
    if n_invervals <= 0:
         raise ValueError(f"{n_intervals=} must be a positive integer")    
```

Reply:
Good point (although, I don't know under which conditions there can be zero intervals).

Please check out the new commit, where (0,0) is included in the "candidate" tables. This will be selected in case n_intervals=0, which in turn will lead to error in plt.subplots(). Is that acceptable?

Reply:
I agree that the error might be of a theoretical nature. 
I would slightly prefer to raise a ValueError in `_get_n_axes` as I suggested above, as the error would occur closer to the source. This would also not make it necessary to add (0,0) to the "candidate" tables. 

But if you prefer to handle it this way, that is also fine by me.

Reply:
I agree, it is better to do as you suggest. I've added this check in the latest commit.